### PR TITLE
Brian/um 32 cascading delete

### PR DESCRIPTION
**Pull request by briancao**
Remove cascading deletes.

### CODE REVIEW

**Comment by briancao** [comment]
Missing mySQL scripts.

File: queries/admin/website/deleteWebsite.js [hunk 1]
```diff
@@ -2,10 +2,28 @@ import { prisma, runQuery } from 'lib/db';
...
+      prisma.event_data.deleteMany({
+        where: { event: { session: { website: { website_id } } } },
+      }),
+      prisma.event.deleteMany({
+        where: { session: { website: { website_id } } },
+      }),
+      prisma.event.deleteMany({
+        where: { session: { website: { website_id } } },
+      }),
+      prisma.event.deleteMany({
```
Comment (mikecao) [inline queries/admin/website/deleteWebsite.js:18]: Why is this repeated 3 times?

File: queries/admin/account/deleteAccount.js [hunk 2]
```diff
@@ -2,10 +2,33 @@ import { prisma, runQuery } from 'lib/db';
...
+      prisma.event_data.deleteMany({
+        where: { event: { session: { website: { user_id } } } },
+      }),
+      prisma.event.deleteMany({
+        where: { session: { website: { user_id } } },
+      }),
+      prisma.event.deleteMany({
+        where: { session: { website: { user_id } } },
+      }),
+      prisma.event.deleteMany({
```
Comment (mikecao) [inline queries/admin/account/deleteAccount.js:18]: Why is this repeated 3 times?

Comment (briancao) [reply queries/admin/account/deleteAccount.js]: Fixed.

File: queries/admin/website/deleteWebsite.js (hunk 1 above)
Comment (briancao) [reply queries/admin/website/deleteWebsite.js]: fixedFixed.

**Pull request merged**